src/lib/validation/utils/validateDatepicker.js

```diff
@@ -2,11 +2,19 @@ import { toggleErrorClass } from './helpers'
 
 export default function validateDatepicker (input) {
   let isValid = true
+
   if (input.value === '') {
-    isValid = false
+    isValid = !input.hasAttribute('required')
   } else {
-    // Regex to check date from https://stackoverflow.com/a/15504877
-    const regex = /^(?:(?:31(.)(?:0?[13578]|1[02]))\1|(?:(?:29|30)(.)(?:0?[13-9]|1[0-2])\2))(?:(?:1[6-9]|[2-9]\d)?\d{2})$|^(?:29(.)0?2\3(?:(?:(?:1[6-9]|[2-9]\d)?(?:0[48]|[2468][048]|[13579][26])|(?:(?:16|[2468][048]|[3579][26])00))))$|^(?:0?[1-9]|1\d|2[0-8])(.)(?:(?:0?[1-9])|(?:1[0-2]))\4(?:(?:1[6-9]|[2-9]\d)?\d{2})$/
+    /**
+     * DpDatePicker accepts the dateFormat: 'dd.mm.yyyy'
+     * Regex to check date from https://www.regextester.com/97612
+     * Allows dates following these rules:
+     * - days may have leading zeros. 1-31. max 2 digits
+     * - months may have leading zeros. 1-12. max 2 digits
+     * - years 1900-2099. 4 digits
+     */
+    const regex = /^\s*(3[01]|[12][0-9]|0?[1-9])\.(1[012]|0?[1-9])\.((?:19|20)\d{2})\s*$/
```

Comment:
While narrowing down the possible input to meet the Datepicker capabilities is fine (reads: no blocker here), i feel like we should possibly allow users to input dates as they like.

Another question: how do we educate users about the expected date format? the datepicker lib does not offer a placeholder, so...

Reply:
A solution could be to reformat the date before passing it to the datepicker. But that would be a story by its own.

Reply:
thanks for adding the tooltip. for now this is ok. in the long run we should display the expected format permanently, as it could easily missed in a tooltip.